pyomo/contrib/fbbt/fbbt.py

```diff
@@ -869,7 +897,7 @@ def fbbt_con(con, deactivate_satisfied_constraints=False, integer_tol=1e-5, infe
 
     # check if the constraint is always satisfied
     if deactivate_satisfied_constraints:
-        if lb >= _lb and ub <= _ub:
+        if lb >= _lb - infeasible_tol and ub <= _ub + infeasible_tol:
```

Comment:
Doesn't this make the `infeasible_tol` effectively twice the value that it is?

Reply:
I don't think so. This tests the lower bound of the constraint and the upper bound of the constraint independently. It ensures that, even after the constraint is deactivated, the body of the constraint will not violate the lower bound of the constraint by more than ```feasibility_tol```. Similarly, even after the constraint is deactivated, the body of the constraint will not violate the upper bound of the constraint by more than ```feasibility_tol```.